tests/test_main.py

```diff
@@ -0,0 +1,24 @@
+import os
+import time
+from logger import Logger, LOG_FOLDER, LOG_EXTENSION
+
+class Test:
+    """
+    Implements logs
+    """
+    def __init__(self, **kwargs):
+        self.ref_test_name = kwargs.get('test_name')
```

Comment:
If this param needs to be present, you could just declare it in the child classes as `self.test_name`.